File: fetch_lottie.py
import urllib.request, json, re
req = urllib.request.Request('https://lottiefiles.com/free-animation/little-power-robot-BtnSKUJQgN', headers={'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64)'})
try:
    html = urllib.request.urlopen(req).read().decode('utf-8')
    matches = re.findall(r'https://[^"\'\s]+\.json|https://[^"\'\s]+\.lottie', html)
    print(list(set(matches)))
except Exception as e:
    print(e)

File: fetch_lottie.js
const https = require('https');
const fs = require('fs');

const options = {
  hostname: 'lottiefiles.com',
  path: '/free-animation/little-power-robot-BtnSKUJQgN',
  method: 'GET',
  headers: {
    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64)'
  }
};

const req = https.request(options, (res) => {
  let data = '';
  res.on('data', (chunk) => {
    data += chunk;
  });
  res.on('end', () => {
    fs.writeFileSync('lottie.html', data);
    console.log('saved');
  });
});
req.end();
